index.html

```diff
@@ -410,6 +216,36 @@ <h2>About me</h2>
         <hr class="mobile-hr">
       </div>
     </div>
+
+    <div class="popup-window" id="pop-window"></div>
```

Comment:
Nice job with the popup window 👍. Now let's try to make it `popup` window centered currently it's off to the left. And also you display the popup window let's try to make the background blur to stick with the Figma design. You can refer to [this link](https://developer.mozilla.org/en-US/docs/Web/CSS/filter-function/blur) for how to do it.

| Your Design | Figma Design |
|---------|----------|
| <img width="700" alt="image" src="https://user-images.githubusercontent.com/34578631/162396580-349040cd-f924-4bed-a572-729355f6c056.png"> | <img width="700" alt="image" src="https://user-images.githubusercontent.com/34578631/162396745-bbec3081-1c63-4597-9d82-6235bb0eaa8a.png"> |

There is unnecessary white space at the bottom of the `popup` window in both mobile and desktop views let's remove that to stick more like to the Figma design.